src/applications/rated-disabilities/components/RatedDisabilityListItem.jsx

```diff
@@ -1,41 +1,30 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+const getHeadingText = ratedDisability => {
+  const { name, ratingPercentage } = ratedDisability;
+
+  const headingParts = [name];
+  if (typeof ratingPercentage === 'number') {
+    headingParts.unshift(`${ratingPercentage}%`);
```

Comment:
I see `ratingPercentage` as a string in some mock data, so if it's a string, it won't get the percentage sign added?

Reply:
I’ll go see what EVSS returns. It should always either be null, or a value (might be a string, not sure)

Reply:
It should always be either null or a number, but I'm just going to check if it's not null to be safe. The mockdata should probably be updated